### DISCUSSION TITLE
Rebroadcast Stream with Motion Events

### DISCUSSION THREAD

**Discussion by ronaldheft** [koush/scrypted#922] [Q&A]
I have a couple of WiFi cameras that I've connected to Scrypted directly using their native camera plugin (Amcrest). This works wonderfully for Scrypted and configuring HKSV. However, I also have a NVR on my Synology using the Surveillance Station software.

To reduce my WiFi traffic, I configured Surveillance Station to connect to the cameras via Scypted's RTSP rebroadcast URL. This saves the extra direct connect to the cameras and reducing the amount of traffic going over my WiFi network. This configuration is solid with one exception; Surveillance Station no longer receives the motion events from the camera and has to do CPU motion detection on my NAS.

Is there anyway to rebroadcast motion events via Scypted? My understanding is RTSP does not contain motion events, but using something like ONVIF would. Is there a way to get a rebroadcast version the ONVIF motion events alongside the stream from Scypted?

Thanks!

**Comment by michbeck100** [comment]
Would be great if the rebroadcast plugin would also support the ONVIF protocol.

**Reply by gapthorpe** [reply]
UniFi Protect now allows third party ONVIF cameras to be imported, and would be amazing if Scrypted could be used to provide that rebroadcast from non-supported cameras as ONVIF.  Was there any update on this request?

**Reply by jaburges** [reply]
@koush not sure if you've been tracking the Unify protect update, but to @gapthorpe's recent post - Unifi will pull in 3rd party ONVIF cameras. If scrypted offered restreaming in ONVIF users could push any camera to Unifi without the need for multiple dockers (including unifi-cam-proxy). :)

**Reply by ZhongDev** [reply]
Am seconding this as well, would be awesome to connect say a spare camera into unifi. for extra coverage

**Reply by PaulW08** [reply]
Would love to have this feature as well!

**Reply by tellok** [reply]
Would love to have this feature as well!

**Reply by KoSoVaR** [reply]
+1 also looking for this feature.

**Reply by andy-hubb** [reply]
+1 this feature would be very useful

**Reply by thecbucks** [reply]
+1

**Reply by caseyjmorton** [reply]
+1

**Reply by snicker** [reply]
also upvoting this
